source/extensions/omni.isaac.lab/test/assets/test_deformable_object.py

```diff
@@ -0,0 +1,336 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
```

Comment:
should we also have tests for the sim and collision APIs in the data class?

Reply:
added init tests to check the tensors are of the right shape